### PR TITLE
[CI] Automatically add CI label

### PR DESCRIPTION
**Pull request by aimorris**
Automatically adds CI label to PR's that edit the .github folder.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+ci:
```
Comment (ErikSchierboom) [inline .github/labeler.yml]: Should this be?

```suggestion
CI:
```

Comment (aimorris) [reply .github/labeler.yml]: It doesn't really matter. It still works. But I'll change it just because the actual label is capitalised like that.

Comment (ErikSchierboom) [reply .github/labeler.yml]: Okay, good to know! And thanks for the change.

**Comment by iHiD** [comment]
Thanks @aimorris :)

**Pull request merged**